onnxruntime/core/optimizer/selection_and_optimization_func.cc

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "selection_and_optimization_func.h"
+#include "core/graph/graph_utils.h"
+#include "core/framework/compute_capability.h"
+#include "core/optimizer/qdq_transformer/constant_folding_dq_node.h"
+
+namespace onnxruntime {
+
+std::vector<std::unique_ptr<ComputeCapability>> ConstantFoldingDQFuncs::Select(const GraphViewer& graph_viewer,
+                                                                               const KeyValueConfig& config,
+                                                                               const GraphOptimizerRegistry& graph_optimizer_registry) {
+  ORT_UNUSED_PARAMETER(config);
+  ORT_UNUSED_PARAMETER(graph_optimizer_registry);
```

Comment:
nit: can do this will less code
```suggestion
                                                                               const KeyValueConfig& /*config*/,
                                                                               const GraphOptimizerRegistry& /*graph_optimizer_registry*/) {
```